### PR TITLE
Like video functionality

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: src/components/LikedPage/LikedVideoCard.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useHistory } from "../../context/historyContext";
+import { useLikedVideo } from "../../context/likedVideoContext";
+import { useWatchLater } from "../../context/watchLaterContext";
+
+const LikedCard = ({ item, watchLaterVideos, LikedVideos }) => {
+  const [dropdown, setDropdown] = useState(false);
+  const trimHeading = (word, n) => {
```
Comment (foolhardy21) [inline src/components/LikedPage/LikedVideoCard.js]: you can move this function out of component as it doesn't need to get initialized on every render.

**Review by foolhardy21 — APPROVED** [review]
code looks good. left one suggestion.

File: src/components/LikedPage/index.jsx [hunk 2]
```diff
@@ -1,14 +1,61 @@
...
+            LikedVideos.map((item) => (
+              <LikedVideoCard
+                key={item._id}
+                item={item}
+                watchLaterVideos={watchLaterVideos}
+                LikedVideos={LikedVideos}
+              />
+            ))
+          ) : (
+            ""
```
Comment (nayyyhaa) [inline src/components/LikedPage/index.jsx:55]: Suggestion: you can use short-circuiting `(&&)`  if no else condition is there.

**Review by nayyyhaa — APPROVED** [review]
Only one suggestion, rest your code looks good to merge. 💯

**Pull request merged**